browser/static/js/main_zstack.js

```diff
@@ -1260,6 +1141,7 @@ function action(action, info, frame = current_frame) {
         // array of arrays, contains annotation data for frame
         if (payload.imgs.hasOwnProperty('seg_arr')) {
           seg_array = payload.imgs.seg_arr;
+          cursor.segArray = seg_array;
```

Comment:
does the global `seg_array` still need to exist? Can we just directly assign `cursor.segArray = payload.imgs.seg_arr;`?